discord/ext/commands/core.py

```diff
@@ -393,6 +398,8 @@ def _ensure_assignment_on_copy(self, other):
             other._buckets = self._buckets.copy()
         if self._max_concurrency != other._max_concurrency:
             other._max_concurrency = self._max_concurrency.copy()
+        if self.extras != other.extras:
+            other.extras = self.extras.copy()
```

Comment:
This is already handled since it's passed as a kwarg so it has no need being here.

Reply:
I was concerned about the interaction with issue #426 is this is treated as mutable data by the user and this is done only be assignment